src/lib/Server/Controller/ContentType.php

```diff
@@ -560,6 +560,35 @@ public function loadContentTypeFieldDefinition($contentTypeId, $fieldDefinitionI
         throw new Exceptions\NotFoundException("Field definition not found: '{$request->getPathInfo()}'.");
     }
 
+    /**
+     * @throws \EzSystems\EzPlatformRest\Exceptions\NotFoundException
+     */
+    public function loadContentTypeFieldDefinitionByIdentifier(
+        int $contentTypeId,
+        string $fieldDefinitionIdentifier,
+        Request $request
+    ): Values\RestFieldDefinition {
+        $contentType = $this->contentTypeService->loadContentType($contentTypeId, Language::ALL);
```

Comment:
The second parameter is `[]` by default, no need to specify it.
```suggestion
        $contentType = $this->contentTypeService->loadContentType($contentTypeId);
```